substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/phases/IntrinsifyMethodHandlesInvocationPlugin.java

```diff
@@ -434,16 +436,22 @@ public boolean apply(GraphBuilderContext b, ResolvedJavaMethod targetMethod, Rec
 
     @SuppressWarnings("try")
     private void processInvokeWithMethodHandle(GraphBuilderContext b, Replacements replacements, ResolvedJavaMethod methodHandleMethod, ValueNode[] methodHandleArguments) {
+        /*
+         * We don't want to process invokes if bci is not unique.
+         */
+        if (((BytecodeParser) b).hasBciDuplication()) {
+            return;
```

Comment:
We should fail here if we capture this case during analysis.

Reply:
done